src/components/SUE/report/SueReportSend.tsx

```diff
@@ -37,12 +37,19 @@ export const SueReportSend = ({
   const { sueReportScreen = {} } = sueConfig;
   const {
     defaultRating = 0,
+    feedbackFormTexts = {},
     reportSendDone = {},
     reportSendLoading = {},
     showFeedbackSection: feedbackSection
   } = sueReportScreen;
   const { title: loadingTitle = '', subtitle: loadingSubtitle = '' } = reportSendLoading;
   const { title: doneTitle = '', subtitle: doneSubtitle = '' } = reportSendDone;
+  const {
+    feedbackHeader = texts.sue.report.sendReportDone.feedbackHeader,
+    ratingTitle = texts.sue.report.sendReportDone.ratingTitle,
+    messageTitle = texts.sue.report.sendReportDone.messageTitle,
+    messagePlaceholder = texts.sue.report.sendReportDone.messagePlaceholder
+  } = feedbackFormTexts;
```

Comment:
what about setting the defaults already with `feedbackFormTexts = texts.sue.report.sendReportDone` to be able to reduce code here like `const { feedbackHeader, ratingTitle, messageTitle, messagePlaceholder } = feedbackFormTexts;`

Reply:
Makes sense I'm updating